updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+
+* xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+* xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-delete-service[Deleting an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-uninstall[Uninstalling the OpenShift Update Service Operator]
+
+
+[id="updating-clusters-overview-upgrade-channels-and-releases"]
+== Understanding upgrade channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels control update recommendations and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of {product-title} always installs that minor version. In addition to the minor version, you can select from the `candidate`, `fast`, `stable` and `eus` channel tiers to enable an update strategy that suits the needs of your environment. For more information, see the following:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and release paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding fast and stable channel use and strategies]
+* xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
+* xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
+
+[id="updating-clusters-overview-update-cluster-using-web-console"]
+== Updating a cluster using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster using the web console]: You can update an {product-title} cluster by using the web console. For more information, see the following items:
```

Comment:
Now that "You can the update a cluster between minor versions" has been removed from this line, there is still a little disparity between this section and the "Updating a cluster within a minor version using the CLI" section, around whether the updates are in a minor release only or between minor releases also.

To provide some continuity between the sections, you could add an additional sentence in this section like the one in the existing [Updating a cluster using the web console section](https://docs.openshift.com/container-platform/4.6/updating/updating-cluster-within-minor.html), to add clarity about what types of updates you can do by using the web console:

> The following steps update a cluster within a minor version. You can use the same instructions for updating a cluster between minor versions.

It would be useful to clarify this in this intro section, because the CLI intro section is specific to updates within a minor release and the reader might be wondering what the case is for the web console updates. By leaving out any mention of the type of updates that are possible, the reader might be left wondering.

Reply:
Ack'ed

Reply:
@xenolinux you have acknowledged this suggestion but I can't see the addition of the suggested line or something similar. Can you please clarify what your have decided upon? Thanks!

Reply:
sorry, I am not sure how it got undone. I added the suggested lines again.